src/main/resources/db/changelog/changes/v1.0.2/changelog-v1.0.2.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+  <include file="sql/update-list-details-table.sql" relativeToChangelogFile="true"/>
```

Comment:
I don't think you can use `<include>` with a SQL file. I think it needs to be a `<sqlFile>` inside a `<changeSet>` block.

Reply:
@mweaver-ebsco, here I have created another folder called sql inside v1.0.2 to include all the sql files in one folder. And I am just including that file with the path through `<include>`. This is the same thing we had for earlier version.

Reply:
Huh... I had no idea that that worked haha